---
title: "Metabolic and co-expression network-based analyses associated with nitrate response in rice"
date: 2014-11-27
doi: https://doi.org/10.1186/1471-2164-15-1056

publishDate: 2020-12-03

authors: 
- Viktoriya Coneva
-  admin
- Jos√© A Casaretto
- Ashraf El-kereamy
- David R Guevara
- Jonathan Cohn
- Tong Zhu
- Lining Guo
- Danny C Alexander
- Yong-Mei Bi
- others

author_notes:
- "Equal contribution"
- "Equal contribution"

tags:
    - Co-expression network
    - Metabolomics 

publication_types: ["2"]
abstract: "**Background:**
Understanding gene expression and metabolic re-programming that occur in response to limiting nitrogen (N) conditions in crop plants is crucial for the ongoing progress towards the development of varieties with improved nitrogen use efficiency (NUE). To unravel new details on the molecular and metabolic responses to N availability in a major food crop, we conducted analyses on a weighted gene co-expression network and metabolic profile data obtained from leaves and roots of rice plants adapted to sufficient and limiting N as well as after shifting them to limiting (reduction) and sufficient (induction) N conditions.

**Results:**
A gene co-expression network representing clusters of rice genes with similar expression patterns across four nitrogen conditions and two tissue types was generated. The resulting 18 clusters were analyzed for enrichment of significant gene ontology (GO) terms. Four clusters exhibited significant correlation with limiting and reducing nitrate treatments. Among the identified enriched GO terms, those related to nucleoside/nucleotide, purine and ATP binding, defense response, sugar/carbohydrate binding, protein kinase activities, cell-death and cell wall enzymatic activity are enriched. Although a subset of functional categories are more broadly associated with the response of rice organs to limiting N and N reduction, our analyses suggest that N reduction elicits a response distinguishable from that to adaptation to limiting N, particularly in leaves. This observation is further supported by metabolic profiling which shows that several compounds in leaves change proportionally to the nitrate level (i.e. higher in sufficient N vs. limiting N) and respond with even higher levels when the nitrate level is reduced. Notably, these compounds are directly involved in N assimilation, transport, and storage (glutamine, asparagine, glutamate and allantoin) and extend to most amino acids. Based on these data, we hypothesize that plants respond by rapidly mobilizing stored vacuolar nitrate when N deficit is perceived, and that the response likely involves phosphorylation signal cascades and transcriptional regulation.

**Conclusions:**
The co-expression network analysis and metabolic profiling performed in rice pinpoint the relevance of signal transduction components and regulation of N mobilization in response to limiting N conditions and deepen our understanding of N responses and N use in crops."
featured: false
publication: "*BMC Genomics*"
---

